pkg/services/sqlstore/migrations/ualert/alert_rule.go

```diff
@@ -171,9 +177,30 @@ func migrateAlertRuleQueries(data []alertQuery) ([]alertQuery, error) {
 	return result, nil
 }
 
-// fixGraphiteReferencedSubQueries attempts to fix graphite referenced sub queries, given unified alerting does not support this.
-// targetFull of Graphite data source contains the expanded version of field 'target', so let's copy that.
-func fixGraphiteReferencedSubQueries(queryData map[string]json.RawMessage) map[string]json.RawMessage {
+// verifyQuery creates a validation function that checks whether the
```

Comment:
Perhaps the diff is a bit weird bit it seems like the sentence is cut off here 🤔

Reply:
lol